pkg/state/types.go

```diff
@@ -1896,3 +1896,114 @@ func (p *Partial) EqualWithOpts(p2 *Partial, ignoreID, ignoreTS bool) bool {
 	}
 	return reflect.DeepEqual(p1Copy, p2Copy)
 }
+
+// Key represents a key in Kong.
+// It adds some helper methods along with Meta to the original Key object.
+type Key struct {
+	kong.Key `yaml:",inline"`
+	Meta
+}
+
+// Identifier returns the endpoint key name or ID.
+func (v1 *Key) Identifier() string {
+	if v1.Name != nil {
+		return *v1.Name
+	}
+	return *v1.ID
+}
+
+// Console returns an entity's identity in a human
+// readable string.
+func (v1 *Key) Console() string {
+	return *v1.Name
```

Comment:
I see other types sometimes default to ID if name is not available - would it make sense here as well?